UnitTests/ScientistTests.cs

```diff
@@ -138,8 +138,8 @@ public void AnExceptionReportsDuration()
             Assert.Equal(42, result);
             mock.Received().Control();
             mock.Received().Candidate();
-            Result<int> observedResult = TestHelper.Results<int>().First(m => m.ExperimentName == "success");
-            Assert.True(observedResult.Matched);
+            Result<int> observedResult = TestHelper.Results<int>().First(m => m.ExperimentName == "failure");
```

Comment:
Do you think it would be better to use the name of the method as the experiment name to prevent any kind of duplicates in the observed results?